packages/driver/src/cy/commands/navigation.ts

```diff
@@ -973,13 +974,13 @@ export default (Commands, Cypress, cy, state, config) => {
           }
 
           if (options.log) {
```

Comment:
Might be worth changing the check at this point since we're fairly removed from the initial setting of _log.
```suggestion
          if (options._log) {
```

Reply:
`log` and `_log` can be a bit different. 

* `log` is a user-provided or default option. 
* `_log` can be `undefined` if some conditions are met. 

I personally think changing `log` to `_log` doesn't change much in this case, but I believe it's safer to leave it as-is.

Reply:
+1. Agree with @tbiethman, at this point _log is referencing the Log instance created by the command when the user options are `log = true`

Reply:
I changed `log` to `_log` when we're trying to manipulate the `_log`.